app/models/flashcard.rb

```diff
@@ -1,3 +1,5 @@
 class Flashcard < ApplicationRecord
   validates_presence_of :question, :answer
+
+  enum status: [:no_status, :green, :yellow, :red]
```

Comment:
Oliver, could you remind me how enums work in this case is it no_status = 0; green = 1; yellow = 2 & red = 3??

Reply:
correct!